### DISCUSSION TITLE
IOS camera order

### DISCUSSION THREAD

**Discussion by MartinSaaby** [blakeblackshear/frigate#15037] [Ask A Question] [answered]
Hi 
I am using the Frigate url as a webapp on my iPhone and it works great. But I can't figure out how to sort the camera order?

On my desktop I am able to make a group and sort the cameras, but that is not possible on my phone?
Am I missing something?

(I am running v. 0.14.1-f4f3cfa)

**Comment by NickM-27** [comment]
The main dashboard can be sorted in the config. I don't believe the camera groups are sortable on mobile

**Reply by MartinSaaby** [reply]
I think that the sort order in a group is saved in local storage. 

How do I sort the main dashboard in config?

**Reply by NickM-27** [reply, marked as answer]
```
cameras:
  camera_one:
    ui:
      order: 0
```

**Reply by MartinSaaby** [reply]
Thank you! I had only tried with one camera to see if it moved to the top, but it seems like cameras without an order are above.
So now that I’m using sorting on multiple cameras, it works just the way I want it :)
